SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/AppCenter.cs

```diff
@@ -15,431 +15,436 @@
-                    continue;
-                }
-                try
-                {
-                    var serviceInstance = serviceType.GetRuntimeProperty("Instance")?.GetValue(null) as IAppCenterService;
-                    if (serviceInstance == null)
-                    {
-                        throw new AppCenterException("Service type does not contain static 'Instance' property of type IAppCenterService. The service is either not an App Center service or it's unsupported on this platform or the SDK is used from a .NET standard library and the nuget was not also added to the UWP/WPF/WinForms project.");
-                    }
-                    StartService(serviceInstance);
-                    serviceNames.Add(serviceInstance.ServiceName);
-                }
-                catch (AppCenterException e)
-                {
-                    AppCenterLog.Error(AppCenterLog.LogTag, $"Failed to start service '{serviceType.Name}'; skipping it.", e);
-                }
-            }
-
-            // Enqueue a log indicating which services have been initialized
-            if (serviceNames.Count > 0)
-            {
-                if (InstanceEnabled)
-                {
-                    _channel.EnqueueAsync(new StartServiceLog { Services = serviceNames });
-                }
-                else
-                {
-                    if (_startedServiceNames == null)
-                    {
-                        _startedServiceNames = new List<string>();
-                    }
-                    _startedServiceNames.AddRange(serviceNames);
-                }
-            }
-        }
-
-        private void StartService(IAppCenterService service)
-        {
-            if (service == null)
-            {
-                throw new AppCenterException("Attempted to start an invalid App Center service.");
-            }
-            if (_channelGroup == null)
-            {
-                throw new AppCenterException("Attempted to start a service after App Center has been shut down.");
-            }
-            if (_services.Contains(service))
-            {
-                AppCenterLog.Warn(AppCenterLog.LogTag, $"App Center has already started the service with class name '{service.GetType().Name}'");
-                return;
-            }
-            if (!InstanceEnabled && service.InstanceEnabled)
-            {
-                service.InstanceEnabled = false;
-            }
-            service.OnChannelGroupReady(_channelGroup, _appSecret);
-            _services.Add(service);
-            AppCenterLog.Info(AppCenterLog.LogTag, $"'{service.GetType().Name}' service started.");
-        }
-
-        #endregion
-    }
+	/// <summary>
+	/// SDK core used to initialize, start and control specific service.
+	/// </summary>
+	public partial class AppCenter
+	{
+		// Internals for testing
```

Comment:
Formatting is wrong, was that edited on Mac? Should redo the patch or clean it up.

Reply:
Redid the change on windows, fixed in 2be10e